### ISSUE TITLE
Suggestion: Add A Reset Current Map Location Key

### ISSUE THREAD

**Issue by LucieG0** [stardew-access/stardew-access#356]
Just opening the ticket for the resetting current map location key as discussed so that it warps you out from wherever you're stuck and takes you to the entrance of the current map, which would hopefully reset the object tracker.

**Comment by khanshoaib3** [comment]
Like the idea but what could that keybind be? There are already too many keybind imo.
This will basically be mimic `debug warp <current>`, so in a a way there is already a command for it. 
So for a ui alternate, how about using the map page in some way to achieve this?

**Comment by ParadoxiKat** [comment]
You're thinking like a developer, not a blind player 💜
Developers (sighted or blind) are fine with remembering large lists of commands to do things. End users / players typically are put off / overwhelmed by commandline things. Remembering all these commands or looking them up is harder than a hotkey for most. Many blind users struggle to use NVDA to it's fullest extent and don't know how to review the text output in the smapi window.
Furthermore, blind screen reader users are familiar with using hotkeys to do basically everything. This is how we expect to interact with things. There will be exceptions of course but I believe a vast majority of blind players would prefer a large list of hotkeys they have to remember or look up over commands with arguments and switches that they have to enter manually in the cli or in the chatbox even.
There are plenty of keys on a keyboard -- and for those who find this one useless they can unassign it or never use it... and just use the warp command.

I think a good key can just be ctrl+shift+w, or alt+shift+w (for warp). Not being used by anything else at the moment.

**Comment by khanshoaib3** [comment]
@ParadoxiKat I agree with the keybind solution but IMO, allowing teleporting with a keybind that is very easy to access is too easy to access, even if we're teleporting to the current map. It's possible that someone triggers the keybind by mistake and then they get teleported. I propose limiting access to the keybind in the map page or something.
Oh and I feel like we should prevent this functionality in mines, it will make escaping from monsters easy.
